### DISCUSSION TITLE
iOS OctoPod and self-hosted Obico server

### DISCUSSION THREAD

**Discussion by iamkhris** [gdombiak/OctoPod#638] [General]
I hope I'm  just missing something, but I can not find a way to connect OctoPod to a locally hosted instance of Obico that I have running.  Can this be done, or is it not possible?

**Comment by gdombiak** [comment]
Hi @iamkhris,

I was not aware that Obico let's you run your own server. Is this an official feature from them? As you realized, OctoPod does not let you set your own Obico server URL at the moment. If this is an official feature then we can easily add the option to point to your own server.

Gaton

**Comment by iamkhris** [comment]
Hi Gaston,

https://www.obico.io/docs/server-guides/

That is where I started to get my own server running at home.  I’m assuming since the information is hosted on their site, it is an official feature from them, but I could also be very wrong.

Do with this info as you will, and have a great day. 😊

Thanks
Chris


From: Gaston Dombiak ***@***.***>
Date: Sunday, April 2, 2023 at 8:08 PM
To: gdombiak/OctoPod ***@***.***>
Cc: Christopher Phillips ***@***.***>, Mention ***@***.***>
Subject: Re: [gdombiak/OctoPod] iOS OctoPod and self-hosted Obico server (Discussion #638)

Hi @iamkhris<https://github.com/iamkhris>,

I was not aware that Obico let's you run your own server. Is this an official feature from them? As you realized, OctoPod does not let you set your own Obico server URL at the moment. If this is an official feature then we can easily add the option to point to your own server.

Gaton
